python/tvm/_ffi/_ctypes/packed_func.py

```diff
@@ -141,6 +141,17 @@ def _make_tvm_args(args, temp_args):
         elif isinstance(arg, TVMContext):
             values[i].v_int64 = _ctx_to_int64(arg)
             type_codes[i] = TypeCode.TVM_CONTEXT
+        elif isinstance(arg, bytes):
```

Comment:
do you need the separate elif block here? seems like you could just make the following block `elif isinstance(arg, (bytearray, bytes))` and collapse the logic there.

Reply:
Good pt `(ctypes.char * n).from_buffer` only takes bytearray, so in the case of bytes, we have to construct a bytearray and push to the temp. 

In the case of bytearray we can directly use it as it is. We can certainly try to unify a bit by having a wrapper function to further check for bytes. That was the original reason for not unifying the two. I will try to take anoter stab